OpenApiSpecs/exchangeSetService_OpenApi_definition_internal.yaml

```
@@ -0,0 +1,1291 @@
+              datacontenttype: "application/json"
+              data:
+                _links:
+                  exchangeSetBatchStatusUri:
+                    href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272/status"
+                  exchangeSetBatchDetailsUri:
+                    href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272"
+                  exchangeSetFileUri:
+                    href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272/files/exchangeset123.zip"
+                  errorFileUri:
+                    href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272/files/error.txt"
+                exchangeSetUrlExpiryDateTime: "2021-02-17T16:19:32.269Z"
+                requestedProductCount: 4
+                exchangeSetProductCount: 1
+                requestedProductsAlreadyUpToDateCount: 0
+                requestedProductsNotInExchangeSet:
+                  [
+                    { productName: "101GB40079ABCDEFG", reason: "invalidProduct" },
+                  ]
+                fssBatchId: "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272" 
+    ####################################
+    #      Security Objects      #
+    ####################################
+
+  securitySchemes:
+    jwtBearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+
+    ######################
+    #      callbacks     #
+    ######################
+
+  callbacks:
+    fulfilmentResponse:
+      '{$request.query.callbackUri}':
+        post:
+          summary: Notify the Exchange Set requestor that this is now ready to download on the File Share Service.
+          description: |
+            Once the Exchange Set has been committed on File Share Service, a notification will be sent to the callbackURI (if specified).  
+
+            ## Data:
+
+            The data for the notification will follow the CloudEvents 1.0 standard, with the data portion containing the same Exchange Set data as the response to the original API request.
+
+          requestBody:
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/cloudEvent"
+          responses:
+            "200":
+              description: The service will ignore all response from the callback.
+    
+    s100FulfilmentResponse:
+      '{$request.query.callbackUri}':
+        post:
+          summary: Notify the Exchange Set requestor that this is now ready to download on the File Share Service.
+          description: |
+            Once the Exchange Set has been committed on File Share Service, a notification will be sent to the callbackURI (if specified).  
+
+            ## Data:
+
+            The data for the notification will follow the CloudEvents 1.0 standard, with the data portion containing the same Exchange Set data as the response to the original API request.
```

This will be for the s100 Exchange Set response.

```suggestion
            The data for the notification will follow the CloudEvents 1.0 standard, with the data portion containing the same S-100 Exchange Set data as the response to the original API request ( $ref: "#/components/schemas/s100ExchangeSetResponse" ).
```

---

Updated.